sklearn/tests/test_multioutput.py

```diff
@@ -336,3 +341,123 @@ def test_multi_output_exceptions():
     y_new = np.column_stack((y1, y2))
     moc.fit(X, y)
     assert_raises(ValueError, moc.score, X, y_new)
+
+
+def test_classifier_chain_fit_and_predict():
+    """Fit classifier chain and verify predict performance"""
+    X, Y = datasets.make_multilabel_classification(n_samples=10000,
+                                                   n_features=100,
+                                                   n_classes=10)
+    classifier_chain = ClassifierChain(LogisticRegression())
+    classifier_chain.fit(X, Y)
+    Y_pred = classifier_chain.predict(X)
+    Y_pred_binary = (Y_pred >= .5)
+    assert_equal(Y_pred.shape, Y.shape)
+    assert_greater(jaccard_similarity_score(Y, Y_pred_binary), 0.5)
+
+
+def test_classifier_chain_fit_and_predict_with_linear_svc():
+    """Fit classifier chain and verify predict performance using LinearSVC"""
+    X, Y = datasets.make_multilabel_classification(n_samples=10000,
+                                                   n_features=100,
+                                                   n_classes=10)
+    classifier_chain = ClassifierChain(LinearSVC())
+    classifier_chain.fit(X, Y)
+    Y_pred = classifier_chain.predict(X)
```

Comment:
I think it's okay to test `predict`, `predict_proba`, `decision_function` and `coef_` sizes all in one test function.